### ISSUE TITLE
HTTPS requests passing through port 80

### ISSUE THREAD

**Issue by johnbfair** [mulesoft/sqs-connector#50]
I'm experiencing an issue where we're trying to send messages to an SQS queue over HTTPS, but when watching the request with WireShark we're seeing it go over port 80 (confirmed by blocking port 80 and getting errors of timeouts).  Has anyone else ever seen a problem like this?  We're using the latest version of the SQS connector (3.0.0 if memory serves) in the Mule ESB.

**Comment by karthikanantha** [comment]
Hi John,

Have you configured the HTTPS configuration in the HTTP connector and set the protocol to HTTPS in SQS connector? 

Can you paste the code related to the use case that you are trying to evaluate?

**Comment by johnbfair** [comment]
Hey @karthikanantha, thanks for responding!  We were originally getting an error message when trying to deploy to our test environment w/ HTTPS selected.  We made it past that, but ran into a change that happened w/ v3.0 of the connector surrounding the support of the new metadata attributes.  We weren't passing anything, so we were getting null errors.  So we added this back and it works!

```
<sqs:message-attributes ref="#[null]"/>
```

Appreciate the help.  :smile:
